mysql/src/test/scala/zio/sql/mysql/FunctionDefSpec.scala

```diff
@@ -9,7 +9,7 @@ object FunctionDefSpec extends MysqlRunnableSpec with ShopSchema {
   import Customers._
   import FunctionDef._
 
-  val spec = suite("Mysql FunctionDef")(
+  override def specLayered = suite("Mysql FunctionDef")(
```

Comment:
Why this change?

Reply:
to avoid duplicating addCustomLayerShared at the end ;) its not a big deal but well I think it always adds some value.